artman/tasks/gapic_tasks.py

```diff
@@ -116,8 +117,19 @@ def execute(self, language, toolkit_path, descriptor_set, service_yaml,
         if service_yaml:
             args = args + ['--service_yaml=' + os.path.abspath(service_yaml)]
         args = args + gapic_args
+
+        gapic_artifact = ''
+        if aspect == 'ALL':
```

Comment:
It would be nice if `aspect` name (`ALL`) matched `gapic_artifact` (`LEGACY_GAPIC_AND_PACKAGE`). Note, `CODE` and `PACKAGE` already match (`LEGACY_` prefix is fine).

Reply:
The problem is that `aspect` is also used for just PROTOBUF generation - you can do `PROTOBUF` + `CODE`. (on the command line: `artman ... java_proto --aspect CODE`)

Reply:
Then maybe `CODE_AND_PACKAGE` instead of `ALL`? There is no way to findout what `ALL` without reading docs/comments.

Reply:
Then maybe `CODE_AND_PACKAGE` instead of `ALL`? There is no way to findout what `ALL` without reading docs/comments.